tools/create-genesis/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "create-genesis"
+version = "0.1.0"
+authors = ["Cryptape Technologies <contact@cryptape.com>"]
+edition = "2018"
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
+json = "0.11.13"
+serde_json = "1.0.39"
+ethabi = "7.0.0"
+ethereum-types = "0.5.2"
```

Comment:
Is it ok to replace it with `cita-types`?

Reply:
I had tried to remove `ethereum_types` and use `cita_types`, it's failed. `ethereum-types` not compatible with cita-type now

Reply:
ok. 
I've checked other crates. cita-bft and cita-forever still use ethereum-types.
Will replace it together in the future.

Reply:
Replace with `numext` is a good choice.

Reply:
yeah. You can have a try if you want.

I have thought that `cita-types` use `numext` instead of `ethereum-types`.